src/conf_mode/tftp_server.py

```diff
@@ -91,6 +93,9 @@ def generate(tftpd):
         else:
             config['listen_address'] = f'[{address}]:{port} -6'
 
+        if 'vrf' in tftpd['listen_address'][address]:
+            config['vrf'] = tftpd['listen_address'][address]['vrf']
```

Comment:
Please change iterator to iterate over the dictionary using key/value pairs - `for address, address_config in tftp['listen_address'].items()`  to make reading the code easier and not iterate over multiple dictionaries.

Reply:
Done